src/structures/GuildAuditLogs.js

```diff
@@ -384,18 +385,18 @@ class GuildAuditLogsEntry {
       case Actions.CHANNEL_OVERWRITE_UPDATE:
       case Actions.CHANNEL_OVERWRITE_DELETE:
         switch (data.options.type) {
-          case 'member':
-            this.extra = guild.members.cache.get(data.options.id) || { id: data.options.id, type: 'member' };
-            break;
-
-          case 'role':
+          case 0:
```

Comment:
This should really not be hard coded, but used with a TS-enum-like object, basic example here

```js
const OverwriteTypes = {
	0: 'role',
	1: 'member',
	role: 0,
	member: 1
}
```

Reply:
the api seems to return strings here, is there a point making an enum type if i still need to use the literal for it?

Reply:
an obj with a key of `0` (number) is the same as `'0'` (string), would keep things consistent imo, but it's up to you!